ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/TransactionCommitterSpec.scala

```diff
@@ -191,6 +192,15 @@ class TransactionCommitterSpec extends WordSpec with Matchers with MockitoSugar
         }
       }
     }
+
+    "mark config key as accessed in context" in {
```

Comment:
This is a good unit test but I would expect this change to have a higher-level effect on the pre-execution result. Is there a test case we can also add to `CommitterSpec`?

Reply:
Yes, for `Committer` I have added tests covering how the current configuration is being retrieved.

Reply:
OK, awesome.